repos/fdbt-site/src/interfaces/index.ts

```diff
@@ -583,21 +583,13 @@ export interface JourneyPattern {
         Id: string;
         Display: string;
     };
+
     endPoint: {
         Id: string;
         Display: string;
     };
-    stopList: string[];
-}
 
-export type StopPoint = {
```

Comment:
Were these unused?

Reply:
This particular one was, there was the exact same thing defined elsewhere.